packages/components/src/grid/README.md

```diff
@@ -27,56 +27,64 @@ function Example() {
 
 ## Props
 
-##### align
-
-**Type**: `CSS['alignItems']`
+##### `align`: `CSS['alignItems']`
 
 Adjusts the block alignment of children.
 
-##### alignment
+-   Required: No
 
-**Type**: `GridAlignment`
+##### `alignment`: `GridAlignment`
 
 Adjusts the horizontal and vertical alignment of children.
 
-##### columns
+-   Required: No
 
-**Type**: `number`,`(number`,`null)[]`
+##### `columns`: `number | number[]`
 
 Adjusts the number of columns of the `Grid`.
 
-##### gap
+-   Required: No
+-   Default: `2`
```

Comment:
Looks like we're missing the `columnGap` prop in the readme.

Reply:
Will add later today 👍